config/rails.yml

```diff
@@ -0,0 +1,21 @@
+Rails:
+  Enabled: true
+
+Rails/FindEach:
+  Enabled: false
+
+Rails/OutputSafety:
+  Enabled: true
+
+Rails/PluralizationGrammar:
+  Enabled: true
+
+Rails/RequestReferer:
+  Enabled: true
+  EnforcedStyle: referrer
+
+Rails/ScopeArgs:
+  Enabled: true
+
+Rails/UniqBeforePluck:
+  Enabled: true
```

Comment:
I like that extraction of Rails-specific cops 👍 . I wonder if `GitHub/ApplicationRecord` should also be part of this list? Perhaps it could even be renamed to `Rails/ApplicationRecord`? What do you think?